scripts/ai/AIDriveStrategyUnloadCombine.lua

```diff
@@ -51,8 +51,19 @@ expect the combine to know best when it is going to perform some maneuvers.
 This is currently screwed up...
 
 
+---------------------------------------------
```

Comment:
"Unload" is a bit confusing, maybe add a little clarification, listing how this can be loaded, by a harvesting combine or a silo loader (picking up a heap)?